pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -58,6 +58,9 @@ func addDefaultingFuncs(scheme *kruntime.Scheme) error {
 
 func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfiguration) {
 
+	localFeatureGate := utilfeature.DefaultMutableFeatureGate.DeepCopy()
+	_ = localFeatureGate.SetFromMap(obj.FeatureGates)
```

Comment:
Based on prior work I cannot change the signature of `SetDefaults_*` to return this error.... we ok with this?

Reply:
panic()  ?